spec/en/reference.md

```diff
@@ -81,36 +83,62 @@ If an agency collects disaggregate, stop-level ridership data, board_alight.txt
+|   |  | * **0** - No ramp/lift used.  |
+|   |  | * **1** - Ramp/lift deployed.  |
+| ramp_boardings | Optional | The **ramp_boardings** field contains the total count of ramp or lift deployed boardings at the identified **stop_id**. Non-negative integer. |
+| ramp_alightings | Optional | The **ramp_alightings** field contains the number of ramp or lift deployed alightings at the identified **stop_id**. Non-negative integer. |
 | service_date | Optional | The **service_date** field contains the date of the associated boarding and/or alighting data at the identified stop. The format is YYYYMMDD. |
 | service_arrival_time | Optional | The **service_arrival_time** field contains the time of the actual arrival at the identified stop. The format is HH:MM:SS. |
 | service_departure_time | Optional | The **service_departure_time** field contains the time of the actual departure from the identified stop. The format is HH:MM:SS. |
-| schedule_relationship | Optional | The **schedule_relationship** field identifies whether service was scheduled and operated, or scheduled but not operated, or operated but not scheduled. If a trip is added it must have a **trip_id** that is not scheduled to run on that day and is unique among trips added on that day.  |
-|   |  | * **0** - (or empty) Service was scheduled and operated  |
-|   |  | * **1** - whole scheduled trip was cancelled.  |
-|   |  | * **2** - whole scheduled trip was cancelled (but replaced with an added trip.)  |
-|   |  | * **3** - trip ran but this **stop_time** was cancelled.   |
-|   |  | * **4** - trip ran but this **stop_time** was cancelled (but replaced with a different stop.)  |
-|   |  | * **5** - whole trip was added. |
-|   |  | * **6** - whole trip was added (as a replacement.)  |
-|   |  | * **7** - this **stop_time** was added to a scheduled trip.    |
-|   |  | * **8** - this **stop_time** was added to a scheduled trip (replacing something cancelled).    |
 | source | Optional | The **source** field contains the collection method of the associated data. |
 |   |  | * **0** - Manual.  |
 |   |  | * **1** - APC.  |
 |   |  | * **2** - AFC.  |
 |   |  | * **3** - Model estimation.  |
 |   |  | * **4** - Mixed source.  |
 
+### *__trip_capacity.txt__*
+
+File: **Optional**
+
+The trip_capacity.txt file identifies the capacity of a vehicle operating a trip on a given day to support crowding analysis in conjunction with board_alight.txt.
+
+|  Field Name | Required | Details |
+|  ------ | ------ | ------ |
+| agency_id | Optional | The **agency_id** field contains the ID of the agency associated with the capacity data. This value is referenced from the agency.txt file. Use this field when you are providing data from more than one agency.  |
```

Comment:
Is this to be the agency with which the vehicle is associated (e.g. one agency contracts a vehicle from another)? Otherwise, are there ever going to be any cross-agency listings?